l10n_eu_product_adr_report/README.rst

```diff
@@ -0,0 +1,22 @@
+ADR product report
+==================
+
+This module creates a new field in products:
+
+* ADR class field for dangerous goods
+
+
```

Comment:
Kindly add usage section so it will help end user to check on runbot.
https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst